### ISSUE TITLE
kata logs are getting flooded with msg="failed to get OOM event from sandbox"

### ISSUE THREAD

**Issue by chavafg** [kata-containers/kata-containers#395]
# Description of problem

While trying to run e2e tests with kata2.0, disk space (120G) got full because kata logs are being flooded with same message:
```
Jul 08 15:38:13 katav2-e2e kata[16530]: time="2020-07-08T15:38:13.627498066Z" level=warning msg="failed to get OOM event from sandbox" error="rpc error: code = NotFound desc = /grpc.AgentService/GetOOMEvent is not supported" sandbox=e9705e88434a40b345c11d505a42e66fdb2c5953d55901ceeb0d3a9758bbfc65
```

# Expected result

kata journal logs shouldnt get flooded with same message

# Actual result
Logs flooded with same message:

```
Jul 08 15:38:13 katav2-e2e kata[16530]: time="2020-07-08T15:38:13.623617Z" level=warning msg="sandbox's cgroup won't be updated: cgroup path is empty" ID=e9705e88434a40b345c11d505a42e66fdb2c5953d55901ceeb0d3a975
8bbfc65 sandbox=e9705e88434a40b345c11d505a42e66fdb2c5953d55901ceeb0d3a9758bbfc65 source=virtcontainers subsystem=sandbox
Jul 08 15:38:13 katav2-e2e kata[16530]: time="2020-07-08T15:38:13.627498066Z" level=warning msg="failed to get OOM event from sandbox" error="rpc error: code = NotFound desc = /grpc.AgentService/GetOOMEvent is not supported" sandbox=e9705e88434a40b345c11d505a42e66fdb2c5953d55901ceeb0d3a9758bbfc65
Jul 08 15:38:13 katav2-e2e kata[16530]: time="2020-07-08T15:38:13.628021002Z" level=warning msg="failed to get OOM event from sandbox" error="rpc error: code = NotFound desc = /grpc.AgentService/GetOOMEvent is not supported" sandbox=e9705e88434a40b345c11d505a42e66fdb2c5953d55901ceeb0d3a9758bbfc65
Jul 08 15:38:13 katav2-e2e kata[16530]: time="2020-07-08T15:38:13.628496734Z" level=warning msg="failed to get OOM event from sandbox" error="rpc error: code = NotFound desc = /grpc.AgentService/GetOOMEvent is not supported" sandbox=e9705e88434a40b345c11d505a42e66fdb2c5953d55901ceeb0d3a9758bbfc65
Jul 08 15:38:13 katav2-e2e kata[16530]: time="2020-07-08T15:38:13.629533605Z" level=warning msg="failed to get OOM event from sandbox" error="rpc error: code = NotFound desc = /grpc.AgentService/GetOOMEvent is not supported" sandbox=e9705e88434a40b345c11d505a42e66fdb2c5953d55901ceeb0d3a9758bbfc65
Jul 08 15:38:13 katav2-e2e kata[16530]: time="2020-07-08T15:38:13.630319659Z" level=warning msg="failed to get OOM event from sandbox" error="rpc error: code = NotFound desc = /grpc.AgentService/GetOOMEvent is not supported" sandbox=e9705e88434a40b345c11d505a42e66fdb2c5953d55901ceeb0d3a9758bbfc65
Jul 08 15:38:13 katav2-e2e kata[16530]: time="2020-07-08T15:38:13.630782991Z" level=warning msg="failed to get OOM event from sandbox" error="rpc error: code = NotFound desc = /grpc.AgentService/GetOOMEvent is not supported" sandbox=e9705e88434a40b345c11d505a42e66fdb2c5953d55901ceeb0d3a9758bbfc65
Jul 08 15:38:13 katav2-e2e kata[16530]: time="2020-07-08T15:38:13.631336529Z" level=warning msg="failed to get OOM event from sandbox" error="rpc error: code = NotFound desc = /grpc.AgentService/GetOOMEvent is not supported" sandbox=e9705e88434a40b345c11d505a42e66fdb2c5953d55901ceeb0d3a9758bbfc65
Jul 08 15:38:13 katav2-e2e kata[16530]: time="2020-07-08T15:38:13.632651819Z" level=warning msg="failed to get OOM event from sandbox" error="rpc error: code = NotFound desc = /grpc.AgentService/GetOOMEvent is not supported" sandbox=e9705e88434a40b345c11d505a42e66fdb2c5953d55901ceeb0d3a9758bbfc65
Jul 08 15:38:13 katav2-e2e kata[16530]: time="2020-07-08T15:38:13.633076348Z" level=warning msg="failed to get OOM event from sandbox" error="rpc error: code = NotFound desc = /grpc.AgentService/GetOOMEvent is not supported" sandbox=e9705e88434a40b345c11d505a42e66fdb2c5953d55901ceeb0d3a9758bbfc65
Jul 08 15:38:13 katav2-e2e kata[16530]: time="2020-07-08T15:38:13.633417171Z" level=warning msg="failed to get OOM event from sandbox" error="rpc error: code = NotFound desc = /grpc.AgentService/GetOOMEvent is not supported" sandbox=e9705e88434a40b345c11d505a42e66fdb2c5953d55901ceeb0d3a9758bbfc65
Jul 08 15:38:13 katav2-e2e kata[16530]: time="2020-07-08T15:38:13.633784696Z" level=warning msg="failed to get OOM event from sandbox" error="rpc error: code = NotFound desc = /grpc.AgentService/GetOOMEvent is not supported" sandbox=e9705e88434a40b345c11d505a42e66fdb2c5953d55901ceeb0d3a9758bbfc65
Jul 08 15:38:13 katav2-e2e kata[16530]: time="2020-07-08T15:38:13.634605253Z" level=warning msg="failed to get OOM event from sandbox" error="rpc error: code = NotFound desc = /grpc.AgentService/GetOOMEvent is not supported" sandbox=e9705e88434a40b345c11d505a42e66fdb2c5953d55901ceeb0d3a9758bbfc65
Jul 08 15:38:13 katav2-e2e kata[16530]: time="2020-07-08T15:38:13.635018081Z" level=warning msg="failed to get OOM event from sandbox" error="rpc error: code = NotFound desc = /grpc.AgentService/GetOOMEvent is not supported" sandbox=e9705e88434a40b345c11d505a42e66fdb2c5953d55901ceeb0d3a9758bbfc65
Jul 08 15:38:13 katav2-e2e kata[16530]: time="2020-07-08T15:38:13.635475912Z" level=warning msg="failed to get OOM event from sandbox" error="rpc error: code = NotFound desc = /grpc.AgentService/GetOOMEvent is not supported" sandbox=e9705e88434a40b345c11d505a42e66fdb2c5953d55901ceeb0d3a9758bbfc65
Jul 08 15:38:13 katav2-e2e kata[16530]: time="2020-07-08T15:38:13.63632227Z" level=warning msg="failed to get OOM event from sandbox" error="rpc error: code = NotFound desc = /grpc.AgentService/GetOOMEvent is not supported" sandbox=e9705e88434a40b345c11d505a42e66fdb2c5953d55901ceeb0d3a9758bbfc65
Jul 08 15:38:13 katav2-e2e kata[16530]: time="2020-07-08T15:38:13.636938712Z" level=warning msg="failed to get OOM event from sandbox" error="rpc error: code = NotFound desc = /grpc.AgentService/GetOOMEvent is not supported" sandbox=e9705e88434a40b345c11d505a42e66fdb2c5953d55901ceeb0d3a9758bbfc65
Jul 08 15:38:13 katav2-e2e kata[16530]: time="2020-07-08T15:38:13.637517352Z" level=warning msg="failed to get OOM event from sandbox" error="rpc error: code = NotFound desc = /grpc.AgentService/GetOOMEvent is not supported" sandbox=e9705e88434a40b345c11d505a42e66fdb2c5953d55901ceeb0d3a9758bbfc65
Jul 08 15:38:13 katav2-e2e kata[16530]: time="2020-07-08T15:38:13.637970383Z" level=warning msg="failed to get OOM event from sandbox" error="rpc error: code = NotFound desc = /grpc.AgentService/GetOOMEvent is not supported" sandbox=e9705e88434a40b345c11d505a42e66fdb2c5953d55901ceeb0d3a9758bbfc65
Jul 08 15:38:13 katav2-e2e kata[16530]: time="2020-07-08T15:38:13.638344509Z" level=warning msg="failed to get OOM event from sandbox" error="rpc error: code = NotFound desc = /grpc.AgentService/GetOOMEvent is not supported" sandbox=e9705e88434a40b345c11d505a42e66fdb2c5953d55901ceeb0d3a9758bbfc65
Jul 08 15:38:13 katav2-e2e kata[16530]: time="2020-07-08T15:38:13.638648629Z" level=warning msg="failed to get OOM event from sandbox" error="rpc error: code = NotFound desc = /grpc.AgentService/GetOOMEvent is not supported" sandbox=e9705e88434a40b345c11d505a42e66fdb2c5953d55901ceeb0d3a9758bbfc65
Jul 08 15:38:13 katav2-e2e kata[16530]: time="2020-07-08T15:38:13.638969451Z" level=warning msg="failed to get OOM event from sandbox" error="rpc error: code = NotFound desc = /grpc.AgentService/GetOOMEvent is not supported" sandbox=e9705e88434a40b345c11d505a42e66fdb2c5953d55901ceeb0d3a9758bbfc65
Jul 08 15:38:13 katav2-e2e kata[16530]: time="2020-07-08T15:38:13.639350677Z" level=warning msg="failed to get OOM event from sandbox" error="rpc error: code = NotFound desc = /grpc.AgentService/GetOOMEvent is not supported" sandbox=e9705e88434a40b345c11d505a42e66fdb2c5953d55901ceeb0d3a9758bbfc65
```

At some point, after running several pods, the OS disk may get full.

**Comment by liubin** [comment]
This should be fixed by #394 , can you check it again ?

**Comment by chavafg** [comment]
Thanks @liubin, it is fixed, closing.
